tests/TestCase.php

```diff
@@ -2,6 +2,34 @@
 
 namespace ipl\Tests\Web;
 
+use ipl\Html\ValidHtml;
+use PHPUnit\Util\Xml;
+
+// phpcs:disable
+if (class_exists('PHPUnit_Util_XML')) {
+    // Support older PHPUnit versions
+    class_alias('PHPUnit_Util_XML', 'PHPUnit\\Util\\Xml');
+}
+
 abstract class TestCase extends \PHPUnit\Framework\TestCase
 {
+    /** Assert that HTML is equal */
+    protected function assertHtml(string $expectedHtml, ValidHtml $actual): void
```

Comment:
While you're at it, please add a third argument `$message` and pass it on to `assertEquals`